fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/MessageHandler.java

```diff
@@ -37,20 +36,21 @@ public MessageHandler(
   /**
    * Send messages to the corresponding handler.
    *
-   * @param laoRepository the repository to access the messages and LAOs
+   * @param laoRepo the repository to access the messages and LAOs
    * @param messageSender the service used to send messages to the backend
    * @param channel the channel on which the message was received
    * @param message the message that was received
    */
   public void handleMessage(
-      LAORepository laoRepository,
+      MessageRepository messageRepo,
+      LAORepository laoRepo,
```

Comment:
Different naming conventions: **Lao**View vs **LAO**Repository

Reply:
Yes I've noticed that the repository package has a different convention than the rest.
I'll change it in a separate PR if that is ok with you